geomesa-process/src/main/scala/org/locationtech/geomesa/process/Point2PointProcess.scala

```diff
@@ -40,7 +40,10 @@ class Point2PointProcess extends VectorProcess {
                minPoints: Int,
 
                @DescribeParameter(name = "breakOnDay", description = "Break connections on day marks")
-               breakOnDay: Boolean
+               breakOnDay: Boolean,
+
+               @DescribeParameter(name = "filterSingularPoints", description = "Filter on groups that all fall on the same point", defaultValue = "true")
```

Comment:
Is there any situation where we do not want this filtering?  

The ticket implies that without the filtering, exceptions will be thrown.  If that's the case, we don't need to make it configurable...